src/component/BpmnVisu.ts

```diff
@@ -17,25 +17,27 @@ import MxGraphConfigurator from './mxgraph/MxGraphConfigurator';
 import { mxgraph } from 'ts-mxgraph';
 import { defaultMxGraphRenderer } from './mxgraph/MxGraphRenderer';
 import { defaultBpmnParser } from './parser/BpmnParser';
-import { MxGraphFactoryService } from '../service/MxGraphFactoryService';
+import MxClientConfigurator from './mxgraph/MxClientConfigurator';
 
-export default class BpmnVisu {
-  private mxClient: typeof mxgraph.mxClient = MxGraphFactoryService.getMxGraphProperty('mxClient');
-  private mxUtils: typeof mxgraph.mxUtils = MxGraphFactoryService.getMxGraphProperty('mxUtils');
+declare const mxClient: typeof mxgraph.mxClient;
+declare const mxUtils: typeof mxgraph.mxUtils;
 
+export default class BpmnVisu {
   public readonly graph: mxgraph.mxGraph;
 
   constructor(protected container: Element) {
     try {
-      if (!this.mxClient.isBrowserSupported()) {
-        this.mxUtils.error('Browser is not supported!', 200, false);
+      if (!mxClient.isBrowserSupported()) {
+        mxUtils.error('Browser is not supported!', 200, false);
       }
+      // Configure codec in mxClient
```

Comment:
:warning: I suggest to hide this and call it in `MxGraphConfigurator` with something like we do for `ShapeConfigurator.initMxShapePrototype`